### DISCUSSION TITLE
How to change routes with redux in nextjs13?

### DISCUSSION THREAD

**Discussion by JUNERDD** [reduxjs/redux#4603] [Help]
![image](https://github.com/reduxjs/redux/assets/66345767/913a8ae3-83f5-431f-8926-5c89ae4a4941)

**Comment by phryneas** [comment]
This is more of a Next.js usage question - what do their docs say about triggering a navigation outside of a component?

**Reply by JUNERDD** [reply]
They mentioned using redirect(), but it didn't work。What are the best practices if I want to make route changes with redux？

**Reply by markerikson** [reply]
First issue is that you can never call `useRouter` in a thunk, because that's a hook.
